Hazelcast.Test/SimpleMapTestFromClient.cs

```diff
@@ -146,8 +147,8 @@ public static void StatDisplayTask()
 
         private static void Main(string[] args)
         {
+            Environment.SetEnvironmentVariable("hazelcast.logging.level", "Info");
```

Comment:
Was this change supposed to be in this PR?

Reply:
yes, a minor cleanup